src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
According to the checklist, interfaces should not use verbs in their names. A more appropriate name for this interface might be 'AreaComputable' to better adhere to Java naming conventions.